lib/Backends/CPU/libjit/libjit_defs.h

```diff
@@ -24,8 +24,13 @@
 typedef SSIZE_T ssize_t;
 #endif
 
-typedef float float4 __attribute__((ext_vector_type(4)));
-typedef float float8 __attribute__((ext_vector_type(8)));
+#if defined(__clang__)
```

Comment:
let's remove https://github.com/pytorch/glow/blob/master/tests/benchmark/CMakeLists.txt#L1
and let it run on CI.

Reply:
OK, I _think_ what I've done should work; I had to include the CPU backend subdirectory when GLOW_WITH_CPU is off, but nothing should be referencing it except GemmTest.  We'll see what CI says...